include/eve/module/math/regular/pow.hpp

```diff
@@ -30,44 +30,56 @@ namespace eve
 //!   namespace eve
 //!   {
 //!      template< eve::floating_value T, eve::floating_value U >
-//!      T pow(T x, U y) noexcept;
+//!      auto pow(T x, U y) noexcept;                                       //1
+//!
+//!      template< eve::floating_value T, eve::floating_value U >           //2
+//!      auto pow(eve::as_complex_t<T> x, U y) noexcept;
+//!
+//!      template< eve::floating_value T, eve::floating_value U >           //2
```

Comment:
Should be 3 and 4 not 2 and 2

Reply:
OR if they are the same thing, just put  //2 once

Reply:
look at the return value description